service/security/shiro/src/main/java/org/eclipse/kapua/service/authentication/shiro/JwtCredentialsImpl.java

```diff
@@ -19,9 +19,11 @@
     private static final long serialVersionUID = -5920944517814926028L;
 
     private String jwt;
+    private String idToken;
```

Comment:
Hey @gbarbon,
can you change the header of this file to " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates and others" ? 
Because last change is from the current year 😄